### ISSUE TITLE
Inconsistent letter spacing in Link, Text, and Paragraph

### ISSUE THREAD

**Issue by theetrain** [telus/tds-core#779]
## Description

Originally reported by @ah-arch.

- The `<Text />` and `<Paragraph />` components have inconsistent `letter-spacing` and `line-height` values. 
- The `size="base"` prop value for the `<Text />` component has undocumented behaviour 

## Reproduction Steps

- Use `<Text />` and `<Text block />` as itself and within `<Paragraph />`
- Observe inconsistent `letter-spacing` behaviour 
- See screenshot below

## Reproduction Percentage (if applicable)

- 10/10 times

## Meta

- TDS version: All
- Willing to develop solution: No
- Has workaround: No
- High impact: Yes

## Screenshots

![image](https://user-images.githubusercontent.com/12798751/47720808-23f59d80-dc25-11e8-8fd0-546c5fb04805.png)

**Comment by jraff** [comment]
For those QAing:
```
<Text block>Tell us what you think</Text>
<Text size="base">Tell us what you think</Text>
<Text size="medium">Tell us what you think</Text>
<Paragraph>Tell us what you think</Paragraph>
<Box><Link>Tell us what you think</Link></Box>
<Paragraph><Link>Tell us what you think</Link></Paragraph>
```

**Comment by jraff** [comment]
Here's an interesting case

```
<Paragraph size="large">
	Tell us what you think
</Paragraph>
<Paragraph size="large">
    <Text size="base">Tell us what you think</Text>
</Paragraph>
```

![Screen Shot 2019-03-14 at 4 44 42 PM](https://user-images.githubusercontent.com/1036187/54390340-a03ec980-4678-11e9-8b26-7700e8533a92.png)

What purpose does `<Text size="base" />` serve?

**Comment by jraff** [comment]
Fixed in [@tds/core-css-reset@1.2.2](https://github.com/telus/tds-core/releases/tag/%40tds%2Fcore-css-reset%401.2.2). @ah-arch
